axon-framework/deadlines/deadline-managers.md

```diff
@@ -57,6 +57,11 @@ Note that there are more options to cancel a deadline next to the previously men
 > Since the non-Pro version of JobRunr has no way to search for deadlines, besides by id, all of the `cancelAll` methods are not implemented for the `JobRunrDeadlineManager`.
 > The [JobRunr Pro extension](../../../extensions/jobrunrpro.md) does support those but requires the Pro version of JobRunr.
 
+> **Caveats for the DbScheduler implementation.**
+>
+> Db-scheduler has no way to filter out tasks. This means for the implementation of the `cancelAll` methods it will need to serialize all the task data, looping over it. If you have many active deadlines, this might take noticeable time and resources.
```

Comment:
```suggestion
> Db-scheduler has no way to filter out tasks. This means that the `cancelAll` implementation will need to serialize all the task data, looping over it. If you have many active deadlines, this might take noticeable time and resources.
```